scripts/main.py

```diff
@@ -1,6 +1,8 @@
 import modules.scripts as scripts
 import sys
 import importlib
+from torch import multiprocessing
+multiprocessing.set_start_method('spawn')

```

Comment:
does this work if you reload the webui without completely restarting?

Reply:
In case another extension calls this function it will blow up too. May want to consider restoring the previous value as well.

Reply:
you're right that it's not working, I think it's best if we check the value before setting it